src/panoptes/pocs/focuser/focuser.py

```diff
@@ -456,9 +454,11 @@ def _autofocus(self,
             # Fit data around the maximum value to determine best focus position.
             # Initialise models
             shift = models.Shift(offset=-focus_positions[imax])
+            # TODO document magic coefficient numbers.
```

Comment:
You mean the initial values? Basically they are just very small values with the expected sign, to help the fitting set off in the correct direction.